habitat/tasks/nav/nav.py

```diff
@@ -821,6 +824,16 @@ def _draw_shortest_path(
                 self.line_thickness,
             )
 
+    def _is_on_same_floor(
+        self, height, ref_floor_height=None, ceiling_height=2.0
+    ):
+        same_floor = False
+        if ref_floor_height is None:
+            ref_floor_height = self._sim.get_agent(0).state.position[1]
+        if ref_floor_height < height < ref_floor_height + ceiling_height:
+            same_floor = True
```

Comment:
This can be simplified to a  single return True and return False

Reply:
Hi, I have already changed it. Is there something else?